### PR TITLE
Add new accounts in Deckungsbeitragsrechnung 

### PR DESCRIPTION
**Pull request by cristinamghita**

### CODE REVIEW

File: backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5436280_sys_09696_adding_accounts_in_deckungs.sql [hunk 1]
```diff
@@ -1,4 +1,3 @@
-
 CREATE OR REPLACE VIEW report.Direct_Costing_ElementValue_selection AS
 SELECT 
 	e1.C_ElementValue_ID as lvl1_C_ElementValue_ID, e1.value as lvl1_value, e1.name as lvl1_name, ( e1.value || ' ' || e1.name ) as lvl1_label,
```
Comment (metas-ts) [inline backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5436280_sys_09696_adding_accounts_in_deckungs.sql:3]: looks like the change is missing in here :-)

**Review by metas-ts — CHANGES_REQUESTED** [review]
- pls check my comment abou the migration script
- also at https://github.com/metasfresh/sp80/issues/1579#issuecomment-1161959416, FN mentions accounts like 7505 which i also don't see in this PR. Since i lack some understanding it might be fine, but still 🤔

File: backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5436280_sys_09696_adding_accounts_in_deckungs.sql (hunk 1 above)
Comment (cristinamghita) [reply backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5436280_sys_09696_adding_accounts_in_deckungs.sql]: right; edited  the file..

**Comment by cristinamghita** [comment]
> * pls check my comment abou the migration script
> * also at [metasfresh/sp80#1579 (comment)](https://github.com/metasfresh/sp80/issues/1579#issuecomment-1161959416), FN mentions accounts like 7505 which i also don't see in this PR. Since i lack some understanding it might be fine, but still 🤔
 Yes, I did not add it since in the latest comment specified different accounts.

**Review by metas-ts — APPROVED** [review]

**Pull request merged**